src/sctp/mod.rs

```diff
@@ -234,11 +270,15 @@ impl RtcSctp {
 
         match event {
             DatagramEvent::NewAssociation(a) => {
-                info!("New remote association");
-                // Remote side initiated the association
-                self.assoc = Some(a);
-                self.handle = handle;
-                set_state(&mut self.state, RtcSctpState::AwaitAssociationEstablished);
+                if self.assoc.is_some() {
+                    warn!("New remote association even though we already have a local one. Ignoring the remote one.");
+                } else {
```

Comment:
I generally prefer early returns to deep else-statements. I do slip up on this quite often though.